web/config/settings.py

```diff
@@ -310,6 +313,16 @@
 SLACK_CLIENT_ID = None
 
 OCTOPRINT_TUNNEL_CAP = int(os.environ.get('OCTOPRINT_TUNNEL_CAP', '1099511627776'))  # 1TB by default
+OCTOPRINT_TUNNEL_SUBDOMAIN_FMT = '{subdomain_code}.tunnels.{site.domain}'
```

Comment:
This is no longer used, right?

Reply:
Sorry, I cannot see what's the problem. Please fix it if you think there is one.